### PR TITLE
creating social impact relates #59

### PR DESCRIPTION
**Pull request by marwajomaa**

### CODE REVIEW

File: client/src/Components/Pages/SEInformation/SocialImpact/index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import './style.css';
+import Fontawesome from 'react-fontawesome';
+import Select from './../../../CommonComponents/Select';
+import Group1 from './Imgs/Group1.png';
+
+const array = [
```
Comment (thesoorae) [inline client/src/Components/Pages/SEInformation/SocialImpact/index.js]: See comment in other PR https://github.com/FACG4/supply-change/pull/61#discussion_r208784781

File: server/controllers/store_user_data.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const bcrypt = require('bcryptjs');
+const query = require('../database/query/query');
+
+module.exports = (req, res) => {
+  console.log(req.body);
```
Comment (thesoorae) [inline server/controllers/store_user_data.js]: do you need this console log

File: server/controllers/store_user_data.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  const { CompanyHouseNumber,companyName,firstName,lastName,eamil,password,phone,mailingList } = req.body;
```
Comment (thesoorae) [inline server/controllers/store_user_data.js]: spaces between commas

File: server/controllers/store_user_data.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  bcrypt.hash(password, 8, (err, hashedPassword) => {
+    console.log(hashedPassword);
+    if (err) return res.end(JSON.stringify({ err: 'err at hashing password' }));
+    const sql = {
+      text: 'INSERT INTO social_enterprise_basic(SE_name, SE_house_NO,first_name, last_name, email, phone_number,mailing_list, password) VALUES ($1,$2,$3,$4,$5,$6,$7,$8)',
```
Comment (thesoorae) [inline server/controllers/store_user_data.js]: is this safe and sanitized, or can an attacker exploit this with their form inputs? I haven't seen sql queries performed in this way...usually that's handled by controller endpoints. could you point me to some resources about this pattern?

Comment (MohammedYehia) [reply server/controllers/store_user_data.js]: I am not sure what do you mean by another endpoint?
Do you mean another endpoint like another path or separating the query in another file?
I think this point you mentioned is very important and I would be grateful if you could explain more or point me to some resource about this and the security issue related to this topic

**Pull request merged**